build.gradle.kts

```diff
@@ -0,0 +1,221 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import com.github.spotbugs.SpotBugsPlugin
+import com.github.spotbugs.SpotBugsTask
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat
+import versions.*
+
+plugins {
+    java
+    jacoco
+    checkstyle
+    id("org.nosphere.apache.rat") version "0.4.0"
+    id("com.github.ethankhall.semantic-versioning") version "1.1.0"
+    id("com.github.spotbugs") version "1.6.10"
+    witness
```

Comment:
This plugin currently doesn't work. Have you forgotten to commit the classes for `org.apache.jmeter.build.WitnessExtension` and `org.apache.jmeter.build.WitnessPlugin`?

Reply:
Ah, that is funny. We have `ignore **/build/**` rule, and I did not like it very much (as it could match way more directories than expected.

Then I introduced a class `org.apache.jmeter.build.WitnessExtension`. Guess what? It is located in `build` package so it got ignored and it was made invisible to git (so I don't see those files as "uncommitted")